tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -93,6 +93,43 @@
                 NewManager = BaseManager.from_queryset(ModelQuerySet)
                 class MyModel(models.Model):
                     objects = NewManager()
+
+-   case: bug_1785
```

Comment:
I'd prefer more descriptive testcase names, how about...

```suggestion
-   case: from_queryset_custom_manager_subclass
```

Reply:
yes this is better, is have pushed the diff